modules/version-update/pages/update-tool.adoc

```diff
@@ -0,0 +1,538 @@
+= Updating Bonita Runtime 
+:description: This page explains how to update Bonita Runtime version (single or multiple nodes).
+
+This page explains how to update Bonita Runtime version (single or multiple nodes).
+
+[NOTE]
+====
+
+.*Updating* means moving from one Bonita version to a newer one, to benefit from:
+* new features (in Main versions)
+* technology updates (in Main versions)
+* bug fixes (in Maintenance versions)
+ 
+====
+
+== How it works
+
+_Bonita Update Tool_ contains a set of scripts that execute changes on the data for making it compatible with a newer main version of Bonita Runtime. 
+The tool is provided as a zip archive that contains an executable script to determine your current version and request the targeted main Bonita version for the update.
+
+The update process is step-based, with each Bonita version being equal to a step. +
+The script manages the sequence of steps from your current version to the targeted one.
+
+=== Can the update tool be used for upgrade/downgrade between community and subscription?
+The answer is *“No”*. 
+The Bonita Update Tool does not help you upgrade/downgrade between the community/subscription versions, so you cannot xref:upgrade-from-community-to-a-subscription-edition.adoc[change edition] at the same time as updating.
+
+=== Bonita version to target
+Always update to the latest maintenance release version of the chosen main release. Maintenance releases often contain bug fixes made shortly after the release of subsequent main releases.
+____
+*For example:*  Bonita 7.13.2 contains fixes for certain bugs that were found earlier in the 7.13.x versions (like 7.13.0 and 7.13.1). To get all of those fixes, update to the 7.13.2 maintenance version instead of 7.13.1.
+____
+
+[#update]
+
+=== Download
+// add links to Bonita web site and Customer Service center => OK
+Download the target Bonita version bundle and the adequate update tool for your Edition from the http://www.bonitasoft.com/downloads-v2[Bonitasoft web site] for Bonita Community edition or from the https://customer.bonitasoft.com/download/request[Customer Service Center] for Bonita Subscription editions. You will also have to download a Tomcat server that matches the target Bonita Runtime version. 
```

Comment:
The link http://www.bonitasoft.com/downloads-v2 does not exist

Reply:
done.